README.md

```diff
@@ -1,12 +1,24 @@
-# Red Hat AI Accelerator
+# Red Hat Composer AI Infrastructure GitOps
 
-Welcome to the AI Accelerator project source code. This project is designed to initialize an OpenShift cluster with a recommended set of operators and components that aid with training, deploying, serving and monitoring Machine Learning models.
+Welcome to the Red Hat Composer AI Infrastructure GitOps project\! This project is a fork of the upstream [AI Accelerator project](https://github.com/redhat-ai-services/ai-accelerator) and is designed to deploy all of the required cluster components for Composer AI to an OpenShift cluster.
 
-This repo is intended to provide a core set of OpenShift features that would commonly be used for a Data Science environment, but can also be highly customized for specific scenarios. When starting out we recommend making a copy or a fork of this project on your Git based instance, since it utilizes the process of automating IT infrastructure using infrastructure as code and software development best practices such as Git, code review, and CI/CD - known as GitOps.
+![AI Accelerator Overview](documentation/diagrams/AI_Accelerator.drawio.png)
 
-Once the initial components are deployed, several ArgoCD Application objects are created which are then used to install and manage the install of the operators on the cluster.
+## Installation
 
-![AI Accelerator Overview](documentation/diagrams/AI_Accelerator.drawio.png)
+### Prerequisites
+
+- Openshift 16+ cluster
```

Comment:
I think this was meant to be 4.16

Reply:
I was trying to future proof but if you insist